### DISCUSSION TITLE
Generating and exporting logs

### DISCUSSION THREAD

**Discussion by nadrojisk** [cmu-sei/crucible#74] [General]
Does crucible have the ability to export logs generated from the boxes during a simulation?

I recently was using splunk’s attack_range for setting up a cyber range. It has a built into splunk box that all the boxes forward logs to. I didn’t see anything like that in crucibles documentation. Was wondering if that is something that is possible?

**Comment by sei-aschlackman** [comment]
Hi @nadrojisk and thanks for your interest in Crucible. The Crucible software is more of a general platform for deploying many kinds of virtual environments across different hypervisors and clouds and does not include any specific virtual machines built in.

That said, for vmware virtual machines, Crucible's console application does have the ability to download a file from a vm through your web browser that can be accessed through the settings menu when viewing a virtual machine. This requires vmware-tools or open-vm-tools to be installed on the virtual machine, but can be used to pull logs or other types of files from virtual machines within your environment.

**Reply by nadrojisk** [reply]
That makes sense! Thanks for the response.
